src/main/java/worldofzuul/presentation/SolarShopController.java

```diff
@@ -0,0 +1,61 @@
+package worldofzuul.presentation;
+
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.image.ImageView;
+import javafx.scene.input.MouseEvent;
+import javafx.stage.Stage;
+import worldofzuul.Game;
+import worldofzuul.Input.Command;
+import worldofzuul.Input.CommandWord;
+import worldofzuul.Rooms.Shops.Shop;
+
+import java.io.IOException;
+import java.net.URL;
+import java.net.URLDecoder;
+import java.nio.charset.StandardCharsets;
+
+public class SolarShopController {
+    @FXML
+    Label solarPrice1, solarPrice2, solarPrice3;
```

Comment:
Access modifiers

Reply:
They will, we need to hook them up to the price of the solar panels.